pkg/render/fluentd.go

```diff
@@ -427,6 +424,35 @@ func (c *fluentdComponent) envvars() []corev1.EnvVar {
 					},
 				)
 			}
+
+			// When LogTypes is not defined explicitly, the default is to enable
```

Comment:
I think this comment should be removed with the defaulting happening in the controller.